src/main/java/org/example/catcafereservation/ReservationValidator.java

```diff
@@ -0,0 +1,56 @@
+package org.example.catcafereservation;
+
+import org.springframework.context.MessageSource;
+import org.springframework.context.i18n.LocaleContextHolder;
+import org.springframework.stereotype.Component;
+
+import java.time.LocalDate;
+import java.time.LocalTime;
+import java.util.HashMap;
+import java.util.Map;
+
+@Component
+public class ReservationValidator {
+
+    private final MessageSource messageSource;
+
+    public ReservationValidator(MessageSource messageSource) {
+        this.messageSource = messageSource;
+    }
+
+    public void validateReservationNumber(String reservationNumber) {
+        if (reservationNumber == null || reservationNumber.length() != 26) {
```

Comment:
nullの判定ですが、Objectクラスのメソッドを用いた方が可読性が高くてお勧めです。
（これも好みですが...）

```suggestion
        if (Object.isNull(reservationNumber) || reservationNumber.length() != 26) {
```

Reply:
こちら調べたらめちゃくちゃ皆さん推していたので、置き換えました！見た目もよいなと、確かに思いました。
ご確認よろしくお願いいたします 🙇 

[112da37](https://github.com/Ema-Sakai/Assignment-10/pull/6/commits/112da375f36f8491bf707dcc37c043f4a932dc67)